--- i3-4.16.1/libi3/get_config_path.c~	2019-01-27 15:45:19.000000000 +0000
+++ i3-4.16.1/libi3/get_config_path.c	2019-03-29 13:40:07.633479749 +0000
@@ -90,6 +90,12 @@
     free(buf);
 
     /* 4: check the traditional path under /etc */
+    config_path = "/etc/i3/config";
+    if (path_exists(config_path)) {
+        return sstrdup(config_path);
+    }
+
+    /* 4: check the traditional path under /etc */
     config_path = SYSCONFDIR "/i3/config";
     if (path_exists(config_path)) {
         return sstrdup(config_path);
